app/views/cart/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Current Cart</h1>
+
+<% @freelancers.each do |freelancer| %>
```

Comment:
if we push collect_freelancers method to the model, then we will need to alter this maybe @cart.collect_freelancers

Reply:
changed